server/events/unlock_command_runner.go

```diff
@@ -45,6 +48,40 @@ func (u *UnlockCommandRunner) Run(
 	}
 
 	if commentErr := u.vcsClient.CreateComment(baseRepo, pullNum, vcsMessage, command.Unlock.String()); commentErr != nil {
-		ctx.Log.Err("unable to comment: %s", commentErr)
+		ctx.Log.Err("unable to comment on PR %s: %s", pullNum, commentErr)
 	}
+
+	u.commentOnDequeuedPullRequests(ctx, dequeueStatus)
+}
+
+func (u *UnlockCommandRunner) commentOnDequeuedPullRequests(ctx *command.Context, dequeueStatus models.DequeueStatus) {
```

Comment:
Could you add tests for these functions?

Reply:
@nitrocode same issue here, should this function be extracted just for the tests?

Reply:
@nitrocode any thoughts on this?

Reply:
@ghaiszaher Looks like some tests already exist in command_test_runner.go